ofl/notoserif/METADATA.pb

```diff
@@ -7,43 +7,35 @@ fonts {
   name: "Noto Serif"
   style: "normal"
   weight: 400
-  filename: "NotoSerif-Regular.ttf"
-  post_script_name: "NotoSerif"
-  full_name: "Noto Serif"
-  copyright: "Copyright 2012 Google Inc. All Rights Reserved."
+  filename: "NotoSerif[wdth,wght].ttf"
+  post_script_name: "NotoSerif-Regular"
```

Comment:
PS Name change also to coordinate with @ultrasparky

Reply:
@ultrasparky, @davelab6 : what should this be?

Reply:
It should be what you are changing it to, but Adobe Fonts will need to manage the change.

Reply:
I've emailed @ultrasparky and @cjchapman to confirm this, I think we should branch this 1 family off and wait for them to get back to us